SEO/Controllers/VisitController.cs

```diff
@@ -29,14 +29,13 @@ public string Get(int id)
         // POST: api/ValidVisit
         public async Task Post([FromBody]VisitInfo visitInfo)
         {
-            var builderUser = Builders<Visitor>.Filter;
-            var filterUser = builderUser.Eq(x => x.UserInfo.UserName, visitInfo.UserName) & builderUser.Eq(x => x.IPAddress, visitInfo.IPAddress) & builderUser.Eq(x => x.VisitDate, visitInfo.VisitDate);
-            var resultUser = await dataBase.Visitors.Find(filterUser).FirstOrDefaultAsync();
+            var visitFilter = visit.resultVisitor(visitInfo);
```

Comment:
all "dataBase..." calls should be moved to VisitManager. Controller is only a dispatcher. It should just call only one method from VisitManager in this case.